appsettings.Development.json

```diff
@@ -1,10 +1,10 @@
 {
   "Logging": {
     "LogLevel": {
-      "Concept": "Concept",
+      "Concept": "Information",
```

Comment:
Change it to Debug

Reply:
Sure, but I want to change the published file (appsettings.json) to information in all, because in production I think trace is very unecessary